src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -1862,6 +1882,8 @@ def convert_args_to_matter_config(args: argparse.Namespace) -> MatterTestConfig:
     config.app_pid = 0 if args.app_pid is None else args.app_pid
     config.fail_on_skipped_tests = args.fail_on_skipped
 
+    config.legacy = True if args.use_legacy_test_event_triggers else args.use_legacy_test_event_triggers
```

Comment:
isn't this just equivalent to config.legacy = args.use_legacy_test_event_triggers?